tests/menubar-editor/reading-text-color-group-reading.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Reading Text Color group in reading mode</title>
+<link rel="author" title="Valerie Young" href="mailto:valerie@bocoup.com">
+<link rel="help" href="https://w3c.github.io/aria-practices/examples/menubar/menubar-2/menubar-2.html">
+<link rel="help" href="https://w3c.github.io/aria/#menubar">
+<link rel="help" href="https://w3c.github.io/aria/#menu">
+<link rel="help" href="https://w3c.github.io/aria/#menuitem">
+<link rel="help" href="https://w3c.github.io/aria/#menuitemcheckbox">
+<link rel="help" href="https://w3c.github.io/aria/#menuitemradio">
+<link rel="help" href="https://w3c.github.io/aria/#aria-checked">
+<link rel="help" href="https://w3c.github.io/aria/#aria-haspopup">
+<link rel="help" href="https://w3c.github.io/aria/#aria-expanded">
+<link rel="help" href="https://w3c.github.io/aria/#aria-disabled">
+
+<script type="module">
+  import { verifyATBehavior, displayTestPageAndInstructions } from "../resources/aria-at-harness.mjs";
+
+  verifyATBehavior({
+    mode: ["reading"],
+    task: "Reading Text Color group",
+    specific_user_instruction: "read the Text Color group",
+    output_assertions: [
+      [1, "when focus is on a menuitem in Text Color group(Black); read the group(Text Color)"],

```

Comment:
This seems more like an instruction than an assertion. It's possible this should be the "specific_user_instruction"